.github/dependabot.yml

```
@@ -5,6 +5,9 @@ updates:
     directory: /
     schedule:
       interval: weekly
+    ignore:
+      - dependency-name: "actions/checkout"
+        version: "v4"
```

I think we're going to want this to be:

```suggestion
        update-types: ["version-update:semver-patch"]
```